extra/mariabackup/encryption_plugin.cc

```diff
@@ -71,7 +71,16 @@ static std::string get_encryption_plugin_from_cnf()
       plugin_load = line + 12;
       // remote \n at the end of string
       plugin_load.resize(plugin_load.size() - 1);
-      break;
+    }
+
+    if (strncmp(line, "innodb_encrypt_tables=", 22) == 0)
+    {
+      if (!strncmp(line + 22, "ON", 2)
+          || !strncmp(line + 22, "1", 1))
+        srv_encrypt_tables= 1;
+      else if (!strncmp(line + 22, "FORCE", 5)
+               || !strncmp(line + 22, "2", 1))
+        srv_encrypt_tables= 2;
```

Comment:
In the common formatting style, lines should be split after binary operators such as `||`. Only Heikki wanted to split before them.